cognite_toolkit/_cdf_tk/data_classes/_module_toml.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import sys
+from dataclasses import dataclass, field
+from pathlib import Path
+
+if sys.version_info >= (3, 11):
+    import toml
+else:
+    import tomli as toml
+
+
+@dataclass(frozen=True)
```

Comment:
Not sure this needs to be frozen? I did it in ModuleDirectory as that had a lot of cached_properties to avoid having to think about busting caches.